x-pack/solutions/security/plugins/security_solution/public/plugin_contract.ts

```diff
@@ -62,6 +62,7 @@ export class PluginContract {
       getComponents$: this.componentsService.getComponents$.bind(this.componentsService),
       upselling: this.upsellingService,
       onboarding: this.onboardingService,
+      productFeatureKeys$: this.productFeatureKeys$,
```

Comment:
`this.productFeatureKeys$` is a _BehaviorSubject_. Consider exposing an observable instead to the services, so we prevent folks from updating the value.

```suggestion
      productFeatureKeys$: this.productFeatureKeys$.asObservable(),
```